lib/index.js

```diff
@@ -16,19 +16,8 @@ export default class Browsochrones extends WebWorkerPromiseInterface {
   stopTreesLoaded = false
   surfaceLoaded = false
 
-  constructor ({
-    origin,
-    query,
-    stopTrees,
-    transitiveNetwork,
-    webpack = false
-  } = {}) {
+  constructor ({webpack = false} = {}) {
```

Comment:
Yes, the functions in the constructor weren't *guaranteed* to run in order and therefore might not work so they definitely should not have been there.